src/generators/json-simple/index.mjs

```diff
@@ -33,6 +34,10 @@ export default {
     // Gets a remark processor for stringifying the AST tree into JSON
     const remarkProcessor = getRemark();
 
+    // Creates a progress bar for the JSON generation
+    const progressBar = createProgressBar('Generating JSON');
```

Comment:
I think the progress bar should be on the generators.mjs base file (https://github.com/nodejs/api-docs-tooling/blob/main/src/generators.mjs)

So that each generator does not need to increment each piece of the progress bar.